tests/testthat/_snaps/compat-types-check.md

```diff
@@ -95,7 +95,7 @@
     Output
       <error/rlang_error>
       Error in `checker()`:
-      ! `foo` must be a single string, not `""`.
+      ! `foo` must be a single string, not "".
```

Comment:
I guess we aren't going to backtick strings?

Reply:
Double-quoting seems unnecessary.